src/corona_hakab_model/generation/generation_manager.py

```diff
@@ -32,6 +32,12 @@ def __init__(self, circles_consts: CirclesConsts, matrix_consts: MatrixConsts):
         matrix_generation = MatrixGenerator(circles_generation.population_data, matrix_consts=matrix_consts)
         self.matrix_data = matrix_generation.matrix_data
 
+        # save consts to allow export
+        self.circles_consts = circles_consts
+        self.matrix_consts = matrix_consts
+
     def save_to_folder(self, folder):
         self.matrix_data.export(os.path.join(folder, 'matrix_data'))
         self.population_data.export(folder, 'population_data')
+        self.circles_consts.export(folder, 'circles_consts.json')
```

Comment:
Wouldn't it be more appropriate to have export function for each of them instead of bundling them here?